src/MSALWrapper.Benchmark/BrokerBenchmark.cs

```diff
@@ -16,8 +16,8 @@ namespace Microsoft.Authentication.MSALWrapper.Benchmark
     /// </summary>
     public class BrokerBenchmark
     {
-        private readonly Guid clientID = Guid.Parse("00000000-0000-0000-0000-000000000000"); // Replace with your own client ID.
-        private readonly Guid tenantID = Guid.Parse("00000000-0000-0000-0000-000000000000"); // Replace with your own tenant ID.
+        private readonly Guid clientID = Guid.Parse("872cd9fa-d31f-45e0-9eab-6e460a02d1f1"); // Replace with your own client ID.
+        private readonly Guid tenantID = Guid.Parse("72f988bf-86f1-41af-91ab-2d7cd011db47"); // Replace with your own tenant ID.

```

Comment:
Did these GUIDs need to change? Were our benchmarks not working before? 🤔

Reply:
These GUIDs were set as Guid.Empty, because the tenant id and client id are not public.
Now, these two IDs are used in `ado token` and `ado pat`, so I assume we can use them in the benchmark publicly.

But, after consideration, I think this part should be in a separate PR.

Reply:
I think it's actually fine. It's a small enough change and should only impact the local development process. Not gonna block on this.